src/protocols/rdp/keymaps/en_gb_qwerty.keymap

```diff
@@ -0,0 +1,56 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+parent  "base"
+name    "en-gb-qwerty"
+freerdp "KBD_UNITED_KINGDOM"
+
+#
+# Basic keys
+#
+
+map -altgr -shift 0x29 0x02..0x0D      ~ "`1234567890-="
+map -altgr -shift      0x10..0x1B      ~ "qwertyuiop[]"
+map -altgr -shift      0x1E..0x28 0x2B ~ "asdfghjkl;'#"
+map -altgr -shift 0x56 0x2C..0x35      ~ "\zxcvbnm,./"
+
+map -altgr +shift 0x29 0x02..0x0D      ~ "¬!"£$%^&*()_+"
+map -altgr +shift      0x10..0x1B      ~ "QWERTYUIOP{}"
+map -altgr +shift      0x1E..0x28 0x2B ~ "ASDFGHJKL:@~"
+map -altgr +shift 0x56 0x2C..0x35      ~ "|ZXCVBNM<>?"
+
+#
+# Keys requiring AltGr
+#
+
+map +altgr -shift 0x29 ~ "¦"
+map +altgr -shift 0x05 ~ "€"
+map +altgr -shift 0x12 ~ "é"
```

Comment:
I guess technically the below would be handled via Unicode in RDP...

Reply:
They shouldn't be. There is the known issue with dead keys, but that doesn't apply for cases where the desired key can be typed without dead keys. If "é", etc. on a UK keyboard all involve AltGr + some other key, and do not require composition with a dead key, then this will not fall back to Unicode events.